x11-misc/dunst/dunst-1.6.1.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit systemd toolchain-funcs
```

Comment:
We like to have an empty line below EAPI.
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#One_empty_line_below_EAPI.3D_line

Reply:
:P oops, sorry seems I forgot that, will be more careful